sdks/wasm/packager.cs

```diff
@@ -610,6 +619,9 @@ class WasmOptions {
 		ninja.WriteLine ("  description = [IL-LINK]");
 		ninja.WriteLine ("rule aot-dummy");
 		ninja.WriteLine ("  command = echo > aot-dummy.cs; csc /out:$out /target:library aot-dummy.cs");
+		ninja.WriteLine ("rule ilstrip");
+		ninja.WriteLine ("  command = cp $in $out; mono-cil-strip $out");
```

Comment:
Where is `mono-cil-strip`?

Reply:
Shouldn't the `--il-strip` be added to the list of options as well?

Reply:
It is added by this line:
		AddFlag (p, new BoolFlag ("il-strip", "strip IL code from assemblies in AOT mode", opts.ILStrip, b => opts.ILStrip = b));

Reply:
mono-cil-strip is an existing mono tool.

Reply:
Yes see that now.  I thought it was part of the `wasm-tools`.